Climate_police/tests/hw7-mao/us_state_abbrev.py

```diff
@@ -0,0 +1,52 @@
+us_state_abbrev = {
```

Comment:
You may wanna consider the cities instead of the states: it could be less time consuming and much more interesting to visualize data per each city!